Project.toml

```
@@ -1,7 +1,7 @@
 name = "Tables"
 uuid = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 authors = ["quinnj <quinn.jacobd@gmail.com>"]
-version = "1.3.2"
+version = "1.3.3"
```

```suggestion
version = "1.4.0"
```
new features are just a minor bump